exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/ApiController.java

```diff
@@ -414,15 +424,19 @@ private static String hexEncodeTransactionMessage(TransactionMessage transaction
     @VisibleForTesting
     static final String BLOCK_HEIGHT_PARAM = "blockHeight";
     @VisibleForTesting
-    static final String BLOCKCHAIN_BLOCK_PATH = BLOCKCHAIN_ROOT + "/block/:" + BLOCK_HEIGHT_PARAM;
+    static final String BLOCKCHAIN_BLOCK_BY_HEIGHT_PATH = BLOCKCHAIN_ROOT + "/blockByHeight/:"
```

Comment:
I would use request params rather than path variables.
Let's consider next endpoints:
`blockchain/block?height=10`
`blockchain/block?blockId=0xABC`
`blockchain/block/last`
`blockchain/block/transactions?height=10`
`blockchain/block/transactions?blockId=0xABC`